css/navbar.scss

```diff
@@ -42,10 +42,29 @@
   }
 }
 
-@media (prefers-color-scheme: dark) {
-  .navbar {
-    background: #6f6f6f96;
+#region-notice {
+  background: var(--secondary-background);
+  padding: .5em 0;
+  text-align: center;
+  margin: 0 0 3rem 0;
+
+  &:only-child {
+    margin: 0 auto;
+    width: max-content;
   }
+}
+
+#region-notice-close-btn {
+  margin-left: 2rem;
+  color: var(--text-color);
+  border: none;
+  background: none;
+  font-size: 1.5em;
+  font-weight: 700;
+  line-height: 1
```

Comment:
```suggestion
  line-height: 1;
  vertical-align: middle;
```
I think that it looks a bit nicer if the close button is aligned with the text:
Current:
![image](https://user-images.githubusercontent.com/20560218/153027975-6fae4576-1a31-4ffe-b086-8f3f759d9c58.png)

Suggested:
![image](https://user-images.githubusercontent.com/20560218/153028036-e2e0c3e0-e8ae-49fe-b500-bc4e2e685a69.png)